src/components/teacher/ConfirmRepeatablesTable.tsx

```diff
@@ -1,3 +1,4 @@
+import { ThemeProvider } from '@mui/material'
```

Comment:
You might be misusing the MUI theme functionality. `ThemeProvider` is a React Context that usually only needs to be called once at the top level (App.tsx). The theme is then accessed with `useTheme()` hook in a component. You can nest child `ThemeProvider` components inside the parent `ThemeProvider` component, but only if you need to override specific parts of the theme. Go back through where you've used `ThemeProvider` at the child component-level and ask if it's necessary to nest that new `ThemeProvider` inside what's already in `App.tsx`. Basically, "Do I need to override the theme for styling this one specific component because other components of the same type elsewhere in the app will be styled differently?" If not, then you should probably move the theming to the top level `ThemeProvider` and access in components with `useTheme()`. Lmk if you have questions about this

https://mui.com/material-ui/customization/theming/